libs/portals/admin/document-provider/src/queries/index.ts

```diff
@@ -0,0 +1,138 @@
+import { gql } from '@apollo/client'
+
+export const GET_STATISTIC_PROVIDERS_BY_NATIONALID = gql`
+  query StatisticProvidersByNationalId(
+    $input: GetStatisticsProvidersNationalId!
+  ) {
```

Comment:
nitpick: If i remember correctly, the current meta is keeping the graphql files near operation. That is, next to the screen that consumes them.